macros/age_general.sql

```diff
@@ -0,0 +1,11 @@
+{% macro age_general(date_1, date_2 = none) %}
```

Comment:
```suggestion
{% macro calculate_age(date_1, date_2 = none) %}
```

Can you also include a description of what this function does and it's inputs?

Reply:
Changes made!